packages/front-end/components/SchemaBrowser.tsx

```diff
@@ -0,0 +1,247 @@
+import {
```

Comment:
Group all of these components into a directory to keep them organized.  We try not to add files directly in `components` anymore since it got too large to handle.